recipe/meta.yaml

```diff
@@ -172,65 +182,6 @@ outputs:
       doc_url: https://www.prevanders.net/libdwarfdoc/
       dev_url: https://github.com/davea42/libdwarf-code
 
-  - name: libdwarf-testing-meta
```

Comment:
I think this is no longer needed, however I would appreciate confirmation.

Reply:
Agreed, thanks!